### ISSUE TITLE
Neatline plugins should just be available to an exhibit if they've been activated in the Omeka plugins panel

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/Neatline#320]
For all our Neatline widgets/plugins, we have a 3-step process to actually use them:
1. Upload the Neatline widget/plugin to the Omeka plugins directory and activate it in the Omeka Admin panel, like you do Neatline and any other Omeka plugin.
2. Declare that you wish to use that widget/plugin on an individual Neatline exhibit by typing out the widget/plugin name in the _Widgets_ field on the _Exhibit Settings_ form.
3. Select that plugin _again_ under the _Styles_ tab in the Neatline editor.

It seems that step 2 or step 3 could be removed, and step 2 seems the most likely candidate to me.

**Comment by davidmcclure** [comment]
Yeah, this is sort of annoying, but removing them introduces other difficulties we'd have to figure out.

**Step 2** - When someone installs one of the subplugins, it's pretty unlikely that they would want to use it on every single Neatline exhibit. Eg, if you install the Simile extension, you don't want an empty timeline sitting at the bottom of exhibits that don't have any records tagged with dates. If we got rid of the "activation" step at the level of the exhibit, and always included the javascript for all of the installed plugins no matter what, there would have to be logic in each sub-plugin to determine whether it should be instantiated on a given exhibit. So, the Simile plugin would have to do some kind of query on the records to see if any of them have dates, etc. This is doable, but it would add a bit of overhead in the code. It also means that we'd often be including a large amount of unused Javascript on exhibit pages (all of Simile, which is really heavy), even for exhibits that don't use it.

**Step 3** - There are cases where you want a sub-plugin to show some but not all of the records in an exhibit. Eg, with the Waypoints widget, people will often have 20-30 records in an exhibit, but only show 4-5 "header" records on the waypoints stack. We'd lose a lot of flexibility if every record was always displayed on all widgets.
